src/server/master_server/run_master_server_main.cc

```diff
@@ -72,9 +74,18 @@ int main(int argc, char** argv) {
 
   LOG(INFO) << "Server listening on " << server_address;
 
+  // Start the chunk server heartbeat monitor task after services have been
+  // started.
+  auto chunk_servers_heartbeat_task =
+      ChunkServerHeartBeatMonitorTask::GetInstance();
+  chunk_servers_heartbeat_task->Start(config_path);
```

Comment:
Can we directly pass the config here as the ConfigManager is already constructed?

Reply:
done. You're correct. Was just using the existing pattern and was being lazy to change it. I'm getting weak man, can't wait to get this done...lol.